protocol/src/debugProtocol.ts

```diff
@@ -631,6 +678,22 @@ export module DebugProtocol {
 		default?: boolean
 	}
 
+	/** A GotoTarget describes a code location that can be used as a target in the 'goto' request.
+		The possible goto targets can be determined via the 'gotoTargets' request.
+	 */
+	export interface GotoTarget {
```

Comment:
We might want a 'label' string to provide a disambiguation dialog if there is more than one.